containers/unit-tests/run.sh

```diff
@@ -12,6 +12,10 @@ git clone /source /tmp/source
 [ ! -d /source/node_modules ] || cp -r /source/node_modules /tmp/source/
 cd /tmp/source
 
+# HACK: Debugging unit-tests container hanging in Semaphore
+( while true; do sleep 60; ps -xa; done ) &
+trap "kill %%" EXIT
```

Comment:
`sleep5 & echo %%` doesn't work in an interactive shell. Should that maybe be `$!`?

Reply:
I tried ```$!```. It didn't work. I don't care enough to figure out why :)